packages/blocks/src/page-block/default/selection-manager.ts

```diff
@@ -433,12 +400,17 @@ export class DefaultSelectionManager {
   };
 
   private _onContainerMouseMove = (e: SelectionEvent) => {
-    const hoverOption = getHoverBlockOptionByPosition(
+    const hoverOption = getBlockOptionByPosition(
```

Comment:
The concept of `option` looks a bit weird here since `options` should be a static config instead of a dynamically updated state. Other block doesn't have options, and I don't expect a block to be defined together with `options`.

Quick thought: how about renaming `EmbedOption` to `EmbedState` or `EmbedEditingState`? Latter is preferred personally since future third-party blocks could have their own editing states, and concepts could be unified.

Reply:
That’s good idea